ambry-frontend/src/main/java/com/github/ambry/frontend/s3/S3MultipartUploadPartHandler.java

```diff
@@ -180,7 +177,11 @@ private Callback<Void> securityPostProcessRequestCallback(BlobInfo blobInfo) {
     private Callback<String> routerPutBlobCallback(BlobInfo blobInfo) {
       return buildCallback(frontendMetrics.putRouterPutBlobMetrics, blobId -> {
         restResponseChannel.setHeader(RestUtils.Headers.BLOB_SIZE, restRequest.getBlobBytesReceived());
-        idConverter.convert(restRequest, blobId, blobInfo, idConverterCallback(blobInfo, blobId));
+        // TODO [S3] Make changes to sign ETags. Currently they are sent as shown below.
+        //  ETag: {"chunks":[{"blob":"AAYQAQBlAAgAAQAAAAAAAAAAw6UGCoNgS8KGgV-SGXAMdQ","size":4194304},
+        //  {"blob":"AAYQAQBlAAgAAQAAAAAAAAAA6Jaga4MiRFiBXi_jjaiQhg","size":1052262}]}
+        restResponseChannel.setHeader(RestUtils.Headers.LOCATION, blobId);
```

Comment:
What's the reason to skip call to idConverterCallback()?

Reply:
Thanks for the cleanup. 
I guess we can remove "idConverterCallback" function as well.

Reply:
It looks like we have to add an internal header `ambry-internal-key-signed-id-metadata` (as we do in method `PostBlobHandler.setSignedIdMetadataAndBlobSize()` for ID conversion to work. Otherwise, Id conversion seems to be broken. Due to that, I disabled it temporarily. Will add it back in next PR.

@JingQianCloud Since, I will add back IdConvertor in next PR, didn't remove `idConverterCallback` method for now.

Reply:
Got you.